internal/cli/search/index_opts.go

```diff
@@ -170,14 +165,59 @@ func (opts *IndexOpts) NewSearchIndex() (*atlasv2.SearchIndexCreateRequest, erro
 	return i, nil
 }
 
+func (opts *IndexOpts) readCreateFromFile() (*atlasv2.SearchIndexCreateRequest, error) {
+	index := &atlasv2.SearchIndexCreateRequest{}
+	if err := opts.loadStrict(index); err != nil {
+		return nil, err
+	}
+	if err := file.Load(opts.Fs, opts.Filename, index); err != nil {
+		return nil, fmt.Errorf(failedToLoadIndexMessage, err.Error())
+	}
+
+	if index.Type == nil {
+		index.Type = pointer.Get(DefaultType)
+	}
+
+	return index, nil
+}
+
+func (opts *IndexOpts) loadStrict(v interface{}) error {
+	file, err := opts.Fs.Open(opts.Filename)
```

Comment:
`file` conflicts with the imported package name `"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/file" `